taxonomy-editor-frontend/src/pages/allentries/index.jsx

```diff
@@ -21,7 +21,10 @@ import ISO6391 from 'iso-639-1';
 const Entry = () => {
     const title = "Test";
     const { data: nodes, isPending, isError, isSuccess, errorMessage } = useFetch(`${API_URL}nodes`);
-
+    const languageOptions = []
+    Object.keys(IsoCodes).forEach(function(key){
+        languageOptions.push(key)
+    })

```

Comment:
@shubham-1806 I think it's not required to add a new variable and push all the keys here. You can directly use `Object.keys()` to get a list of keys. Slo, this part can be removed.